pallets/task/src/tests.rs

```diff
@@ -316,531 +346,496 @@ fn task_can_be_updated_only_by_one_who_created_it(){
 	});
 }
 
 #[test]
 fn start_tasks_assigns_new_current_owner(){
 	new_test_ext().execute_with( || {
-
 		// Profile is necessary for task creation
-		assert_ok!(Profile::create_profile(Origin::signed(1), username(), interests(), HOURS, Some(additional_info())));
-		assert_ok!(Profile::create_profile(Origin::signed(2), username(), interests(), HOURS, Some(additional_info())));
+		assert_ok!(Profile::create_profile(Origin::signed(*ALICE), username(), interests(), HOURS, Some(additional_info())));
+		assert_ok!(Profile::create_profile(Origin::signed(*BOB), username(), interests(), HOURS, Some(additional_info())));
 
 		// Ensure new task can be created.
-		assert_ok!(Task::create_task(Origin::signed(1), title(), spec(), BUDGET, get_deadline(), attachments(), keywords()));
+		assert_ok!(Task::create_task(Origin::signed(*ALICE), title(), spec(), BUDGET, get_deadline(), attachments(), keywords(), None));
 
 		// Ensure new task is assigned to new current_owner (user 1)
-		let hash = Task::tasks_owned(1)[0];
-		let task = Task::tasks(hash).expect("should found the task");
-		assert_eq!(task.current_owner, 1);
-		assert_eq!(Task::tasks_owned(1).len(), 1);
+		let task_id = Task::tasks_owned(*ALICE)[0];
+		let task = Task::tasks(task_id).expect("should found the task");
+		assert_eq!(task.current_owner, *ALICE);
+		assert_eq!(Task::tasks_owned(*ALICE).len(), 1);
 
 		// Ensure task is started by new current_owner (user 2)
-		assert_ok!(Task::start_task(Origin::signed(2), hash));
+		assert_ok!(Task::start_task(Origin::signed(*BOB), task_id));
 
 		// Ensure when task is started user1 has 0 tasks, and user2 has 1
-		assert_eq!(Task::tasks_owned(1).len(), 0);
-		assert_eq!(Task::tasks_owned(2).len(), 1);
-
+		assert_eq!(Task::tasks_owned(*ALICE).len(), 0);
+		assert_eq!(Task::tasks_owned(*BOB).len(), 1);
 	});
 }
 
 #[test]
 fn start_tasks_assigns_task_to_volunteer(){
 	new_test_ext().execute_with( || {
-
 		// Profile is necessary for task creation
-		assert_ok!(Profile::create_profile(Origin::signed(1), username(), interests(), HOURS, Some(additional_info())));
+		assert_ok!(Profile::create_profile(Origin::signed(*ALICE), username(), interests(), HOURS, Some(additional_info())));
 
 		// Ensure new task can be created.
-		assert_ok!(Task::create_task(Origin::signed(1), title(), spec(), BUDGET, get_deadline(), attachments(), keywords()));
+		assert_ok!(Task::create_task(Origin::signed(*ALICE), title(), spec(), BUDGET, get_deadline(), attachments(), keywords(), None));
 
 		// Ensure new task is assigned to new current_owner (user 1)
-		let hash = Task::tasks_owned(1)[0];
-		let task = Task::tasks(hash).expect("should found the task");
-		assert_eq!(task.current_owner, 1);
-		assert_eq!(Task::tasks_owned(1).len(), 1);
+		let task_id = Task::tasks_owned(*ALICE)[0];
+		let task = Task::tasks(task_id).expect("should found the task");
+		assert_eq!(task.current_owner, *ALICE);
+		assert_eq!(Task::tasks_owned(*ALICE).len(), 1);
```

Comment:
I think we are using the aliases in DAO. Its much more readable than using numbers as accounts. Its a good improvement.

Reply:
Yeah, I copied it from the Dao pallet.